### PR TITLE
Psetup

### PR DESCRIPTION
**Pull request by assadounto**
### Summary
Added contact last part, contact form please review my contact form .
added form validation for name field and email field
required input is email
### Requirement
Name field accepts up to 30 chars.
Email field accepts only emails.
Message field accepts up to 500 chars.
All fields have placeholder text.
All fields have the correct type.
A user cannot submit the form if one or more of the fields is empty.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -129,45 +145,100 @@ <h2 class="card-title">Tonic</h2>
...
+              action="https://formspree.io/f/xjvlpydr"
+              class="form"
+              method="post"
+            >
+              <input
+                type="text"
+                name="user_name"
+                id="user_name"
+                placeholder="Full name"
+                maxlength = "20"
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly be noticed the the max-length for the name must be 30 and it should have a required attribute.

File: index.html [hunk 2]
```diff
@@ -129,45 +145,100 @@ <h2 class="card-title">Tonic</h2>
...
+                required
+              />
+              <textarea
+                name="message"
+                id="message"
+                cols="30"
+                rows="10"
+                placeholder="Enter Text Here"
+                maxlength="30"
+              ></textarea>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly be noticed that the max-length for the textarea should be 500 and it should have a required attribute.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @assadounto , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
#Status: Approved 🟢🟢🟢 

Hi @assadounto,

Good job adding the missing requirement as requested by the previous reviewer. 👍 Keep rocking 💪💪💪 

CompleteDerekMullerGIF
 
Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## To Highlight:
✅ No Linter error
✅ Project is working fine

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**